src/seedu/addressbook/parser/Parser.java

```diff
@@ -41,6 +31,9 @@
                     + " (?<isAddressPrivate>p?)a/(?<address>[^/]+)"
                     + "(?<tagArguments>(?: t/[^/]+)*)"); // variable number of tags
 
+    public static final Pattern PERSON_CREATE_ARGS_FORMAT =
+            Pattern.compile("(?<username>[^/]+)"
```

Comment:
Line 35 should be on the same line as 34. In this case, there is no need to split the line into 2 unless it is more than 110 chars.